src/components/Editor.jsx

```diff
@@ -15,6 +15,8 @@ class Editor extends React.Component {
       this._editor.setValue(nextProps.source);
     }
 
+    //this disables autoclosing of HTML tags
```

Comment:
Rather than putting a comment here, maybe it would be worth just creating a separate instance method e.g. `this._disableAutoClosing()` whose name documents its behavior? I usually like to avoid inline comments as they tend to get out of sync with the thing they’re describing : )